mart/nn/nn.py

```diff
@@ -153,7 +161,11 @@ def forward(self, *args, **kwargs):
         selected_args = [kwargs[key] for key in arg_keys[len(args) :]]
         selected_kwargs = {key: kwargs[val] for key, val in kwarg_keys.items()}
 
-        ret = self.module(*args, *selected_args, **selected_kwargs)
+        try:
```

Comment:
You may remove `try...except` if you want to add the error message in a separate PR.